src/app/explore-page/explore-page.component.scss

```diff
@@ -26,75 +32,90 @@
 }
 
 .cards-container {
-    display: block;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    flex: 1 1 275px;
+    row-gap: 20px;
+    column-gap: 20px;
     padding: 1vw;
 }
 
 .game-card {
     height: 400px;
-    width: 250px;
+    width: 275px;
+    border: 1px solid rgb(197, 193, 193);
+    border-radius: 10px;
+    padding: 0px 2px;
 }
-
 .game-card:hover {
     box-shadow: 0 12px 18px -6px black;
     -webkit-box-shadow: 0 12px 18px -6px black;
 }
-.card-header {  
-    background-color: white;
-    height: 50px;
-    padding: 1vh;
-    margin: 1vh;
+.game-card_header {
+    height: 13%;
+    margin: 1%;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+}
+.game-card_avatar {
+    height: 40px;
+    width: 40px;
+    background-color: $raiting-orange;
+    border-radius: 50%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
 }
-
-.card-avatar {
-    background-color: orangered;
+.game-card_rating {
     color: white;
-    text-align: center;
+    font-family: 'segoe ui';
```

Comment:
If you are using only one font family across app you can use global styling or create variables to handle additional families